include/rebar3_sbom.hrl

```diff
@@ -1,10 +1,55 @@
+%--- Macros --------------------------------------------------------------------
+
 -define(APP, "rebar3_sbom").
 -define(DEFAULT_OUTPUT, "./bom.[xml|json]").
 -define(DEFAULT_VERSION, 1).
 -define(PROVIDER, sbom).
 -define(DEPS, [lock]).
 -define(SPEC_VERSION, <<"1.6">>).
 
+%--- Types ---------------------------------------------------------------------
+
+-type bom_ref() :: string().
+-type spdx_licence_id() :: string().
```

Comment:
ToDo: enumerate the valid strings of spdx_licence. Can be added later, but maybe a comment here for now that the type spec can be further specified.